plugin-info-vue-full-app/README.md

```diff
@@ -0,0 +1,105 @@
+
+```
+yarn install
+```
+
+For `npm` users, install all dependencies using:
+
+```
+npm install
+```
+
+## Build Process
+
+There are two ways to build the plugin for use in Adobe Photoshop:
+
+- `yarn watch` or `npm run watch` will build a development version of the plugin, and recompile everytime you make a change to the source files. The result is placed in `dist`.
+- `yarn build` or `npm run build` will build a production version of the plugin and place it in `dist`. It will not update every time you make a change to the source files.
+
+> You **must** run either `watch` or `build` prior to trying to use within Photoshop!
+
+## Launching in Photoshop
+
+You can use the UXP Developer Tools to load the plugin into Photoshop.
+
+If the plugin hasn't already been added to your workspace in the UXP Developer Tools, you can add it by clicking "Add Plugin..." and selecting `dist/manifest.json`. **DO NOT** select the `manifest.json` file inside the `plugin` folder.
+
+Once added, you can load it into Photoshop by clicking the ••• button on the corresponding row, and clicking "Load". Switch to Photoshop and you should see the starter panel.
+
+## What this plugin does
+
+This plugin mainly provides information on any installed UXP or CEP plugins. For UXP plugins, all relevant store information is gathered and displayed. For CEP plugins, only the installed version and plugin type are shown (this is due to the limited information given in the plugin manifest).
+
+## How to use this plugin
+
+The usage of this plugin is fairly straightforward after the initial install. Once loaded through the UXP Developer Tool, click on `Load Plugins` to go through the first time configuration process. The setup will guide you through adding both your UXP and CEP plugin paths. For reference, they can usually be found in the following directories:
+
+#### Windows
+
+| Type  | Plugin Path                                                 |
+| ----- | ----------------------------------------------------------- |
+| `UXP` | `%appdata%\Adobe\UXP\Plugins\External`                      |
+| `CEP` | `C:\\Program Files (x86)\Common Files\Adobe\CEP\extensions` |
+
+#### macOS
+
+| Type  | Plugin Path                                         |
+| ----- | --------------------------------------------------- |
+| `UXP` | `~/Library/Application Support/Adobe/UXP/Plugins`   |
+| `CEP` | `/Library/Application Support/Adobe/CEP/extensions` |
+
+Once both directories have been selected, click `Finish Setup` to complete the setup process. Afterwards, both your UXP and CEP plugins will be automatically loaded. Simply click on the plugin name in the list to view more information about the plugin.
+
+At any point in time you can redo the setup process by either clicking on `Reload Plugins` in the home menu or using the plugin command called `Show Setup`. Any stored data can also be deleted by using the `Clear Data` plugin command.
+
+## The code explained
```

Comment:
Let me know your thoughts on this. Since this is larger than a regular sample, I thought a section explaining some of the code would be pretty useful